src/InnoSetup/Environment.iss

```diff
@@ -403,9 +386,6 @@ begin
     Log('Cloning IDF: ' + CmdLine);
     DoCmdlineInstall(CustomMessage('DownloadingEspIdf'), CustomMessage('UsingGitToClone'), CmdLine);
 
-    if IDFTempPath <> '' then
```

Comment:
I think you can also delete this code a few lines above:
```
    if IDFTempPath <> '' then
      CmdLine := CmdLine + ' --reference ' + IDFTempPath;
```

because IDFTempPath is never set in the `if NeedToClone` branch.